notebooks/COS/ViewData/ViewData.ipynb

```diff
@@ -0,0 +1,1452 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will import the following packages:\n",
+    "\n",
+    "- `numpy` to handle array functions (version $\\ge$ 1.17)\n",
+    "  - We also insert a line telling numpy to ignore a certain type of warning about divide-by-zeros. This is not always a good idea, so seriously consider it before copying it into your code!\n",
+    "- `astropy.io fits` for accessing FITS files\n",
+    "- `astropy.table Table` for creating tidy tables of the data\n",
+    "- `astropy.units` and `astropy.visualization.quantity_support` for dealing with units\n",
+    "- `matplotlib.pyplot` for plotting data\n",
+    "- `astroquery.mast Mast and Observations` for finding and downloading data from the [MAST](https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html) archive\n",
+    "\n",
+    "*Later on, we will import some functions from a local file*, for which we have imported the `sys` package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This line is just to make matplotlib look good in a notebook\n",
+    "%matplotlib inline\n",
+    "# Manipulating arrays\n",
+    "import numpy as np\n",
+    "# Our arrays will encounter divide by 0 warnings, which is OK, but we want to suppress the warning text with the next code line.\n",
+    "   # SO DO NOT COPY THIS NEXT LINE INTO YOUR CODE WITHOUT CAREFUL CONSIDERATION!\n",
+    "np.warnings.filterwarnings('ignore')\n",
+    "# Reading in data\n",
+    "from astropy.io import fits\n",
+    "from astropy.table import Table\n",
+    "# Dealing with units and plotting them\n",
+    "from astropy import units as u\n",
+    "from astropy.visualization import quantity_support\n",
+    "quantity_support()\n",
+    "# Plotting\n",
+    "import matplotlib.pyplot as plt\n",
+    "# Downloading data from archive\n",
+    "from astroquery.mast import Mast\n",
+    "from astroquery.mast import Observations\n",
+    "from astroquery.mast import Catalogs\n",
+    "\n",
+    "import sys # We need import to look in the directory above this one when we look for some functions later.\n",
+    "sys.path.append('../') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will also define a few directories in which to place our data and plots, as well as a few colors we will use in plots later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# These will be important directories for the notebook\n",
+    "cwd = !pwd\n",
+    "cwd = cwd[0]\n",
+    "\n",
+    "!mkdir ./data\n",
+    "!mkdir ./output/\n",
+    "!mkdir ./output/plots\n",
+    "\n",
+    "datadir = cwd + '/data/'\n",
+    "outputsdir = cwd + '/output/'\n",
+    "plotsdir = cwd + '/output/plots/'\n",
```

Comment:
This section has an issue that it will only work on unix-like systems.  So windows users are just out of luck.  That's not the end of the world, but as an alternative I suggest using the python stdlib `pathlib` functionality: 
```
from pathlib import Path

curdir = Path('.')

datadir = curdir / 'data'

datadir.mkdir()
```

and you can then use `str(datadir)` to get the path string or in many places just use `datadir` directly. And this will work transparently on windows without any weird finaglings.